### DISCUSSION TITLE
Welcome to FIWARE-Big-Bang Discussions!

### DISCUSSION THREAD

**Discussion by fisuda** [lets-fiware/FIWARE-Big-Bang#258] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dkastl** [comment]
Thank you for setting up Discussions! That's a great idea!

I'm involved in several Open Source projects, and I support Open Data and civic technology initiatives.
I'm still learning about FIWARE, but my plan is to use FIWARE in combination with a location-based task management project: https://gtt-project.org/ 

My current research is about data models and whether it's better to focus on NGSI-LD or NGSI-v2. And I'm trying to figure out, how to best connect the GTT-Project with a Context Broker. 

I'm curious how others see a task management application fit into the FIWARE ecosystem, what use cases exist and if there are already similar projects.
